### ISSUE TITLE
Build fails using Qt5.14.2-MSVC2017-64bit

### ISSUE THREAD

**Issue by shujaatak** [Waqar144/CuteMarkEd-NG#2]
This repository seems awesome but as mentioned in the issue's title, it fails to build.

**Comment by shujaatak** [comment]
I like the way this project shows the line numbers so that's why I am interested in this project. I don't know whether QOwnNotes supports this feature or not.

**Comment by Waqar144** [comment]
> This repository seems awesome but as mentioned in the issue's title, it fails to build.

Whats the error msg? I can't help without that.

MSVC support may not be there at all, and i have never built this on windows myself. Can you try with MinGW?

QOwnNotes doesn't have line numbering.

**Comment by shunf4** [comment]
Managed to fix MSVC build process, and added installer generation process: #3
Here is the win32 installer built from my forked repository: https://ci.appveyor.com/project/shunf4/cutemarked-ng/build/artifacts

**Comment by Waqar144** [comment]
Thanks!

I can't test myself since I don't have windows :laughing: @shujaatak can you give it a try? If all goes well we can close the issue

**Comment by shunf4** [comment]
I suggest adding this project to AppVeyor and updating the build status badge in README😄

**Comment by Waqar144** [comment]
> I suggest adding this project to AppVeyor and updating the build status badge in READMEsmile

Yes, I have it on the list, but I have never done it before so I am being lazy about it :laughing:
